addon/-private/engine-ext.js

```diff
@@ -34,73 +25,11 @@ Engine.reopen({
   },
 
   buildRegistry() {
-    let namespace = this;
     let registry = this._super(...arguments);
 
     if (!(this instanceof Ember.Application)) {
-      registry.optionsForType('component', { singleton: false });
-      registry.optionsForType('view', { singleton: false });
-      registry.optionsForType('template', { instantiate: false });
-
-      registry.register('application:main', namespace, { instantiate: false });
-
-      registry.register('controller:basic', Ember.Controller, { instantiate: false });
-
-      // registry.register('renderer:-dom', { create() { return new Renderer(new DOMHelper()); } });
-
-      registry.injection('view', 'renderer', 'renderer:-dom');
-
-      registry.register('template:-outlet', topLevelViewTemplate);
-      registry.injection('route', '_topLevelViewTemplate', 'template:-outlet');
-      registry.register('view:-outlet', OutletView);
-
-      //registry.register('-view-registry:main', { create() { return {}; } });
-
-      registry.injection('view', '_viewRegistry', '-view-registry:main');
-
-      registry.register('view:toplevel', EmberView.extend());
-
-      // registry.register('route:basic', Route, { instantiate: false });
-      // registry.register('event_dispatcher:main', EventDispatcher);
-
-      // registry.injection('router:main', 'namespace', 'application:main');
-      registry.injection('view:-outlet', 'namespace', 'application:main');
-
-      // registry.register('location:auto', AutoLocation);
-      // registry.register('location:hash', HashLocation);
-      // registry.register('location:history', HistoryLocation);
-      // registry.register('location:none', NoneLocation);
-
-      registry.injection('controller', 'target', 'router:main');
-      registry.injection('controller', 'namespace', 'application:main');
-
-      // registry.register('-bucket-cache:main', BucketCache);
-      registry.injection('router', '_bucketCache', P`-bucket-cache:main`);
-      registry.injection('route', '_bucketCache', P`-bucket-cache:main`);
-      registry.injection('controller', '_bucketCache', P`-bucket-cache:main`);
-
-      registry.injection('route', 'router', 'router:main');
-
-      registry.register('component:-text-field', TextField);
-      registry.register('component:-text-area', TextArea);
-      registry.register('component:-checkbox', Checkbox);
-      // registry.register('view:-legacy-each', LegacyEachView);
       registry.register('component:link-to', LinkComponent);
```

Comment:
Can you rename this one?  I keep getting confused if it is the normal `Ember.LinkComponent` or our custom scoped one.  Perhaps you can name the imported variable `EngineScopedLinkComponent`?